test/test-debugletapi.js

```diff
@@ -53,7 +53,7 @@ describe('Debuglet API', function() {
   });
 
   it('should acquire the project number during init', function(done) {
-    debugletapi.init('uid123', { warn: function() {} }, function(err) {
+    debugletapi.init('uid123', { warn: function() {} }, function(err, project) {
       assert(!err);
```

Comment:
Currently this tests whether the project number was retrieved successfully by checking for a lack of errors. It will provide more confidence if we add an assert checking the value of `project` as well now that it is available.